### ISSUE TITLE
ErrImagePull for all images in mindarodev.azurecr.io

### ISSUE THREAD

**Issue by AliakseiVenski** [hsubramanianaks/b2k-samples#1]
Any image in .yml deployment manifests cannot be pulled from registry mindarodev.azurecr.io

![image](https://github.com/user-attachments/assets/28d8838d-57ed-4c0a-a9f5-80db211d0f5c)

**Comment by AliakseiVenski** [comment]
@hsubramanianaks could you help there?

**Comment by mkocaks** [comment]
@AliakseiVenski Did you have any luck with this ?

**Comment by AliakseiVenski** [comment]
@mkocaks Nope, for my PoC with BtK I built all images from dockerfiles and pushed them to our own registry. We chose another solution over BtK, so I don't need this anymore.
